### PR TITLE
Update some Case Studies mentioning Docker

### LINKED ISSUE #31424: Make small wording adjustments to some Case Studies for mentions of docker
ref: https://github.com/kubernetes/website/issues/30934

Some of the case studies need a bit of rewording to remove unnecessary mention of docker.

**Note:** the case studies are HTML + CSS rather than markdown - you need to be familiar with that in order to update these!

https://kubernetes.io/case-studies/peardeck/index.html
➤ I'd reword “Docker containers” to “containers”

https://kubernetes.io/case-studies/prowise/index.html
➤ I'd reword “Dockerize” to “containerize”

https://kubernetes.io/case-studies/squarespace/index.html
➤ I'd reword “Docker file” to “Dockerfile”

https://kubernetes.io/case-studies/wink/index.html
➤ I'd change “just over two years ago” to “in 2015”
➤ I'd also change “Dockerized” to “containerized”

/priority backlog
/triage accepted

### PR DESCRIPTION
**Pull request by Babapool**
Fixes #31424 

In this PR, small wording adjustments are made in the Case Studies that mention `Docker`. These changes include:

- Reword `Docker containers` to `containers`
- Reword `Dockerize` to `containerize`
- Reword `Docker file` to `Dockerfile`
- Changes `just over two years ago` to `in 2015`

Files updated are:

- https://github.com/kubernetes/website/blob/main/content/en/case-studies/peardeck/index.html
- https://github.com/kubernetes/website/blob/main/content/en/case-studies/prowise/index.html
- https://github.com/kubernetes/website/blob/main/content/en/case-studies/squarespace/index.html
- https://github.com/kubernetes/website/blob/main/content/en/case-studies/wink/index.html

### CODE REVIEW

**Comment by mehabhalodiya** [comment]
/lgtm
Awaiting another sig docs reviewer/approver to sign off these changes.

**Comment by reylejano** [comment]
This PR fixes https://github.com/kubernetes/website/issues/31424
/approve

**Comment by Babapool** [comment]
Thanks @reylejano
